### ISSUE TITLE
Warnings from ansible 2.3

### ISSUE THREAD

**Issue by RadekBiba** [RedHatQE/rhui3-automation#61]
The following warnings are printed when ansible 2.3 is used with rhui3-automation:

```
< TASK [rhua : install rhui-installer] >
 --------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

changed: [ec2-54-74-204-139.eu-west-1.compute.amazonaws.com]
 __________________________________________
< TASK [rhua : call rhui installer if nfs] >
 ------------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: {{install_rhui_installer|changed and 'NFS' in groups and
groups['NFS']|length > 0 }}

changed: [ec2-54-74-204-139.eu-west-1.compute.amazonaws.com]
 ______________________________________________
< TASK [rhua : call rhui installer if gluster] >
 ----------------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: {{install_rhui_installer|success and 'GLUSTER' in groups and
groups['GLUSTER']|length > 0 }}

skipping: [ec2-54-74-204-139.eu-west-1.compute.amazonaws.com]

```

**Comment by alexxa** [comment]
@RadekBiba that should be fixed by commit 62913fb on master

**Comment by RadekBiba** [comment]
The warnings are gone, thanks!
